src/clr-angular/data/datagrid/_datagrid.clarity.scss

```diff
@@ -422,6 +422,8 @@
         flex: 1 1 auto;
         align-self: flex-start;
         display: flex;
+        // Keep default cursor - change only if needed
+        cursor: default;
```

Comment:
@bdryanovski I think this isn't fixing the culprit of the problem, but it's trying to suppress/override the symptom. For some reason, we have `pointer` style for some Datagrid buttons which of course should have `pointer` cursor by default. So there was no need to have the `pointer` style in the first place. I tracked where this `pointer` style was coming from, and I found that it's in the `clr-no-styles-button()` mixin in `_helpers.clarity.scss`. The solution is quite possibly removing it from there. But this pointer style was introduced by this [PR](https://github.com/vmware/clarity/pull/3133) which tried to solve another problem. So this issue needs broader investigation.

Reply:
Agree this is not quite the right fix. The button reset we probably still want to have a pointer but since we use the reset on non button elements as well its causing the above issue. A possible fix is to scope the pointer to only apply to button elements on the reset.

```scss
@mixin clr-no-styles-button {
  margin: 0;
  padding: 0;
  border: none;
  border-radius: 0;
  box-shadow: none;
  background: none;
  
  @at-root button#{&} {
    cursor: pointer;
  }
}
```

This will allow the reset to be used on any element but apply the proper cursor on buttons.